model/OAuthClient.php

```diff
@@ -125,7 +127,13 @@ public function send(RequestInterface $request, array $options = [], $repeatIfUn
                 if (json_last_error() != JSON_ERROR_NONE || !is_array($params)) {
                     $params = [];
                 }
-                $response = $this->request($request->getMethod(), $request->getUri(), $params, false);
+
+                $response = $this->request(
+                    $request->getMethod(),
+                    $request->getUri(),
+                    array_merge($options, $params),
```

Comment:
Are we sure we want to send all previous `$options`? Shall we add explicitly specific `$options` keys only?

Reply:
I believe we should send all the options, as that is a request to the same endpoint with the same data, extended by newer access token only;